app/models/pet.rb

```diff
@@ -39,6 +40,7 @@ class Pet < ApplicationRecord
   validates :name, presence: true
   validates :birth_date, presence: true
   validates :breed, presence: true
+  validates :is_published, inclusion: [true, false]
```

Comment:
https://thoughtbot.com/blog/how-to-validate-the-presence-of-a-boolean-field-in-a-rails-model